packages/dkim-mirage/dkim-mirage.0.1.0/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+maintainer:   "Romain Calascibetta <romain.calascibetta@gmail.com>"
+authors:      "Romain Calascibetta <romain.calascibetta@gmail.com>"
+homepage:     "https://github.com/dinosaure/ocaml-dkim"
+bug-reports:  "https://github.com/dinosaure/ocaml-dkim/issues"
+dev-repo:     "git+https://github.com/dinosaure/ocaml-dkim.git"
+doc:          "https://dinosaure.github.io/ocaml-dkim/"
+license:      "MIT"
+synopsis:     "Implementation of DKIM in OCaml"
+description: """DKIM implementation in OCaml"""
```

Comment:
this description is superfluous compared to the `synopsis`. I would suggest to either remove it or expending the description a bit